### PR TITLE
SST and SA options

### PR DESCRIPTION
**Pull request by pcarruscag**

### CODE REVIEW

File: _docs_v7/Physical-Definition.md [hunk 1]
```diff
@@ -97,3 +100,58 @@ The reference values $$\rho_{ref}, T_{ref}, v_{ref}$$ equal the initial state va
...
 **Note:** The initial state is also used as boundary conditions for `MARKER_FAR`.
 
+## Turbulence Models ##
+
+| Solver | Version |
+| --- | --- |
+| `*_RANS` | 7.4.0 |
+
+This section describes how to setup turbulence models for RANS simulations. In general turbulence models are selected via the option `KIND_TURB_MODEL`, corrections/versions and parameters of the models are specified via the options listed below.
+The turbulent Pradtl number can be modified with option `PRANDTL_TURB` (the default is 0.9).
```
Comment (bigfooted) [inline _docs_v7/Physical-Definition.md:110]:
```suggestion
This section describes how to setup turbulence models for RANS simulations. The turbulence model is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
A turbulence model can then be selected via the option `KIND_TURB_MODEL`
Different submodels and parameters are specified via the different options listed below.
The turbulent Prandtl number can be modified with the option `PRANDTL_TURB` (the default is 0.9).
```

File: _docs_v7/Physical-Definition.md [hunk 2]
```diff
@@ -97,3 +100,58 @@ The reference values $$\rho_{ref}, T_{ref}, v_{ref}$$ equal the initial state va
...
+
+### Spalart-Allmaras (SA) ###
+
+SU2 implements several versions and corrections of the SA model.
+The model is selected using `KIND_TURB_MODEL= SA` and the modifications via the `SA_OPTIONS` list, if this list is empty SU2 defaults to `SA-noft2`.
```
Comment (bigfooted) [inline _docs_v7/Physical-Definition.md:115]:
```suggestion
The model is selected using `KIND_TURB_MODEL= SA` and the modifications via the `SA_OPTIONS` list. If this list is empty, then SU2 defaults to `SA-noft2`.
```

File: _docs_v7/Physical-Definition.md [hunk 3]
```diff
@@ -97,3 +100,58 @@ The reference values $$\rho_{ref}, T_{ref}, v_{ref}$$ equal the initial state va
...
+
+The rough wall correction is implicitly turned on by specifying roughness values for wall markers via the `WALL_ROUGHNESS` option.
+
+### Shear Stress Transport (SST) ###
+
+SU2 implements the "Standard" (1994) and 2003 versions of the SST model along with several modifications.
+
+**Note:** Currently all versions are "modified" i.e. the turbulence kinetic energy (TKE) is not included in the viscous stress tensor.
+
+The model is selected using `KIND_TURB_MODEL= SST` and the modifications via the `SST_OPTIONS` list, if this list is empty SU2 defaults to `SSTm` (see warning below).
```
Comment (bigfooted) [inline _docs_v7/Physical-Definition.md:140]:
```suggestion
The main model is selected using `KIND_TURB_MODEL= SST` and the version and modifications via the `SST_OPTIONS` list. If this list is empty SU2 defaults to the baseline 1994 model, `SSTm` (see warning below).
```

File: _docs_v7/Physical-Definition.md (hunk 1 above)
Comment (bigfooted) [reply _docs_v7/Physical-Definition.md:110]: The "Prandtl" remark is a bit out of place... maybe first mention all KIND_TURB_MODEL and XXX_OPTIONS and then other shared options for turbulence?

**Review by bigfooted — APPROVED** [review]
LGTM

File: _docs_v7/Physical-Definition.md [hunk 4]
```diff
@@ -79,7 +82,7 @@ For all schemes, as reference values for the density and temperature the free-st
 
 ## Flow Condition (Incompressible) ##
 
-| Solver | Version | 
+| Solver | Version |
 | --- | --- |
 | `INC_EULER`, `INC_NAVIER_STOKES`, `INC_RANS` | 7.0.0 |
```
Comment (WallyMaier) [inline _docs_v7/Physical-Definition.md:87]: Do these 7.0.0 need to be updated?

Comment (pcarruscag) [reply _docs_v7/Physical-Definition.md:87]: The info there is still valid for v7, what is missing here that I noticed are the NEMO solvers

File: _docs_v7/Physical-Definition.md [hunk 5]
```diff
@@ -97,3 +100,61 @@ The reference values $$\rho_{ref}, T_{ref}, v_{ref}$$ equal the initial state va
 
 **Note:** The initial state is also used as boundary conditions for `MARKER_FAR`.
 
+## Turbulence Models ##
+
+| Solver | Version |
+| --- | --- |
+| `*_RANS` | 7.4.0 |
+
+This section describes how to setup turbulence models for RANS simulations. The turbulence model is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:109]:
```suggestion
This section describes how to setup turbulence models for RANS simulations. Turbulence is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
```

**Pull request merged**